packages/mocktail/README.md

```diff
@@ -216,3 +216,40 @@ To address this, we must explicitly stub `sleep` like:
 final person = MockPerson();
 when(() => person.sleep()).thenAnswer((_) async {});
 ```
+
+#### Why is my method throwing a `TypeError` when stubbing using `any()`?
+
+[Relevant Issue](https://github.com/felangel/mocktail/issues/162)
+
+By default when a class extends `Mock` any unstubbed methods return `null`. When stubbing using `any()` the type must be inferable. However, when a method has a generic type argument it might not be able to infer the type and the method would act as if it was unstubbed.
```

Comment:
```suggestion
By default when a class extends `Mock` any unstubbed methods return `null`. When stubbing using `any()` the type must be inferable. However, when a method has a generic type argument it might not be able to infer the type and as a result, the generic would fallback to `dynamic` causing the method to act as if it was unstubbed.
```